assets/javascripts/discourse-sift/lib/report-reason.js.es6

```diff
@@ -0,0 +1,22 @@
+import showModal from "discourse/lib/show-modal";
+import loadScript from "discourse/lib/load-script";
+
+export function showReportReason(reviewable, callback, opts) {
+  opts = opts || {};
+
+  return loadScript("defer/html-sanitizer-bundle").then(() => {
```

Comment:
I do not believe this loadScript call is necessary here actually - you should be able to directly call the contents of the function

Reply:
I copied that from the user notes example.  I left it because we are allowing the user to enter text into the Other field, so based on my assumption of what the script is for I thought it was still a good idea